src/Service/Cases.php

```diff
@@ -53,24 +53,25 @@ public function FindCases(array $query)
     }
 
     /**
-     * Example query:.
+     * Finds cases by properties.
      *
-     * $query = [
+     * Example data:
+     * $data = [
      *   'FieldProperties' => [
      *     [
      *       'InternalName' => 'ows_CaseID',
      *       'Value' => 'GEO-2022-033830'
      *     ],
      *   ],
      *   'CaseTypePrefixes' => ['GEO'],
-     * ]
+     * ];
      */
-    public function FindByCaseProperties(array $query): array
+    public function FindByCaseProperties(array $data): array
```

Comment:
The parameter should not be called `$data`, but rather `$query`. “Data” is used for creating stuff and not for querying.

Reply:
Updated.